app/src/main/res/layout/activity_cadastrarendereco.xml

```diff
@@ -4,6 +4,162 @@
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <!-- text animation-->
+                <com.airbnb.lottie.LottieAnimationView
+                    android:id="@+id/animationenderecoconfirm"
+                    android:layout_width="200dp"
+                    android:layout_height="match_parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:lottie_loop="true"
+                    app:lottie_autoPlay="true"
+                    app:lottie_rawRes="@raw/coffeeloading"/>
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Confirmar"
+                    android:textColor="#fff"
+                    android:textSize="30dp"
+                    android:textStyle="bold"
+                    app:layout_constraintBottom_toBottomOf="@+id/animationenderecoconfirm"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="@+id/animationenderecoconfirm" />
+
+
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+        </androidx.cardview.widget.CardView>
+
+        <!--register later button-->
+        <androidx.cardview.widget.CardView
+            android:id="@+id/edittextenderecodepoisr"
+            android:layout_width="350dp"
+            android:layout_height="50dp"
+            android:layout_margin="30dp"
+            app:cardBackgroundColor="#FFC75A"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/edittextenderecoconfirmar">
+
+            <!--Animation and text base-->
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <!-- text animation-->
+                <com.airbnb.lottie.LottieAnimationView
+                    android:id="@+id/animationenderecodepois"
+                    android:layout_width="200dp"
+                    android:layout_height="match_parent"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:lottie_loop="true"
+                    app:lottie_autoPlay="true"
+                    app:lottie_rawRes="@raw/coffeeloading"/>
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Cadastrar Depois"
+                    android:textColor="#fff"
+                    android:textSize="30dp"
```

Comment:
Mesma situação do texto acima
Utilizar @string/ para referenciar um texto para que manutença futura seja mais pratico e rápido
Utilizar sp ao invés de dp quando for informar o tamanho do texto